content/outcomes/application-development-how-we-work/support-each-other.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "We're Supported"
+weight: 80
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+
+We don't have traditional command-and-control line management structures within Tanzu Labs. **We have coaching managers.** They help our people get to where they want to go in their careers. 
+
+We believe that the most valuable feedback we can get along that journey comes from our peers — the people that we work with day in, day out. **Our managers help us get that feedback from our peers, and encourage us to give those same peers the feedback they need, too.**
+
+Everyone who joins Tanzu Labs has one-one-ones scheduled with their manager. Our managers help us with all of the questions we have about the expectations set forth in these articles, because **we use this very learning series for our own onboarding**. 
```

Comment:
```suggestion
Everyone who joins Tanzu Labs has one-on-ones scheduled with their manager. Our managers help us with all of the questions we have about the expectations set forth in these articles, because **we use this very learning series for our own onboarding**. 
```